src/mame/vtech/geniuscolor.cpp

```diff
@@ -0,0 +1,80 @@
+// license:BSD-3-Clause
+// copyright-holders:
+
+
+/*************************************************************************************************************
+
+    Skeleton driver for VTech Genius Color Pocket / Super Color Pocket / Genio Color Pocket.
+
+    VTech 35-140500-100-203 PCB with MX25L3206E and N25S10 serial ROMs on one side and two globs on the other.
+    Unknown CPU, program ROM seems compressed.
+
+*************************************************************************************************************/
+
+
+#include "emu.h"
+
+#include "screen.h"
+#include "speaker.h"
+
+
+namespace {
+
+
+class geniuscolor_state : public driver_device
+{
+public:
+	geniuscolor_state(const machine_config &mconfig, device_type type, const char *tag) :
+		driver_device(mconfig, type, tag),
+		m_screen(*this, "screen")
+	{ }
+
+	void geniuscolor(machine_config &config) ATTR_COLD;
+
+protected:
+	required_device<screen_device> m_screen;
+
+	uint32_t screen_update_geniuscolor(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
+};
+
+uint32_t geniuscolor_state::screen_update_geniuscolor(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect)
+{
+	return 0;
+}
+
+// 45-keys foldable keyboard, 8 activity buttons, two direction keys (right, left) and home, OK and power buttons.
+INPUT_PORTS_START( geniuscolor )
+INPUT_PORTS_END
```

Comment:
Is the keyboard foldable, or does it slide behind the screen on these?  Pictures look like it’s a “slider” form factor like some smart phones and PDAs used.

Reply:
Yes! Not foldable, but "slider" type. I've changed the text.

Have you seen the ROM? Any idea about the CPU or its encoding?

Thanks!

Reply:
Sorry, I haven’t looked at the ROM dump.